spec/etcd.spec.js

```diff
@@ -73,12 +73,12 @@ describe('Etcd', function () {
     it('should fetch keys', function () {
       return etcd.fetchConfig(config)
         .should.eventually.partiallyEql({
-          a: [ , , '1' ],
-          b: [ , , '2' ],
-          c: [ , , '3' ],
-          d: [ , , '4', '5' ],
-          e: [ , , '6', , 'hello' ],
-          g: [ , , ]
+          a: [, , '1'],
+          b: [, , '2'],
+          c: [, , '3'],
+          d: [, , '4', '5'],
+          e: [, , '6', , 'hello'],
+          g: [, ,] // eslint-disable-line comma-spacing
```

Comment:
Latest `standard` complains about spaces after `,` and also having a space before `]`.